sensehat/src/main/java/com/google/android/things/contrib/driver/sensehat/SenseHat.java

```diff
@@ -23,12 +23,47 @@
  */
 @SuppressWarnings({"unused", "WeakerAccess"})
 public class SenseHat {
-    public static final int I2C_ADDRESS =  0x46;
-    public static final String BUS_DISPLAY = "I2C1";
+    public static final int I2C_DISPLAY_ADDRESS =  0x46;
```

Comment:
We probably should not rename existing public constants as that will break the API for developers.